.github/workflows/ci-disable-gtest.yml

```diff
@@ -47,17 +47,16 @@ jobs:
 
       - name: Prepare libavif (cmake)
         run: >
-          cmake -G Ninja -S . -B build
+          cmake -S . -B build
```

Comment:
The `CMAKE_BUILD_TYPE` variable is not used by a multi-config generator such as Visual Studio. I remember `-DCMAKE_BUILD_TYPE=Release` will cause a cmake warning to be emitted if the Visual Studio generator is used.

Reply:
I went with `cmake --build build --config Release`. It does not seem to print a warning for single-config generators.

The alternative is specifying `cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_CONFIGURATION_TYPES=Release` everywhere but that will always emit a warning.